src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -250,10 +271,16 @@ public void deployHA() {
     HaHeightOne = Constants.HaHeightMax;
     HaHeightTwo = Constants.HaHeightMax;
     m_HaOne.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
-    sbHaHeightOne.setNumber(HaHeightOne);
 
     m_HaTwo.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
-    sbHaHeightTwo.setNumber(HaHeightOne);
+  }
+
+  public void unlatchHA() {
```

Comment:
Is this used at all?

Reply:
No, since our PID isn't tuned correctly
Removing it